purview/api/v1/__init__.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-3.0+
+from flask import Blueprint
+
+from purview.api.v1.error import (bad_request, not_found, ValidationError)
+
+api_v1 = Blueprint('api_v1', __name__)
```

Comment:
We won't gain anything by having blueprints for our app since there aren't sections of our Flask app (we only have an API). This adds complexity that we don't need.